### ISSUE TITLE
ETA for stable 2.0 

### ISSUE THREAD

**Issue by OsirisTerje** [gsscoder/commandline#328]
Are there any chance of getting a stable version of the 2.0 out?

**Comment by nemec** [comment]
Only @gsscoder has access to do that and he currently has other commitments to attend to. If you're concerned about crashing the latest master is stable, but unfortunately it's not available via NuGet.

**Comment by OsirisTerje** [comment]
It seems stable enough to me, I have been using it for more than 6 months now (the latest beta on nuget) , and it works just fine.  I am mostly concerned that it has a beta version number, and when it is marked as a dependency in another package that package also has to have a beta version number.  So it is mostly annoying, can work around it by not declaring it, but not so nice.  But guess we have to wait on @gsscoder .

**Comment by Mizipzor** [comment]
@nemec do you know the current status on the stable? I have some time to spend on this project right now and I'm really keen to fix the remaining bugs that we would require for stable. But I dont know the roadmap. Even if @gsscoder is busy surely he can just put in a new nuget version if the source is in a good place.

**Comment by nemec** [comment]
You know, I'm not really sure. There is a 2.0 stable milestone, but comments within one of the issues indicate that it's postponed until post-2.0 release. TBH I'm happy to call it stable.

**Comment by zidad** [comment]
I'm running into the same issue as @OsirisTerje, I'd really like a stable release number to be able to use this project in other (non-beta) packages... @gsscoder any chance?

**Comment by igloo15** [comment]
@gsscoder hasn't been active in a long time should someone maybe create a new nuget package of this?

**Comment by zidad** [comment]
@igloo15 @OsirisTerje Funny! I tried to make an unofficial released package called CommandLineParser.Unofficial, but when I tried to push it to nuget I found out someone already had exactly the same idea:  https://www.nuget.org/packages/CommandLineParser.Unofficial

**Comment by nemec** [comment]
I finally got access to the official repo and pushed a new prerelease package based on the latest master. #361 seems to be the only outstanding bug that should be resolved before we can release a stable 2.0.

https://www.nuget.org/packages/CommandLineParser/2.1.1-beta

**Comment by MgSam** [comment]
Any update on when 2.0 will be released? The NuGet publish date on 1.9.71 of  Feb 27, 2013 probably makes a lot of people think this project is dead.

Also- could someone update the CodePlex site so that it says at the top in large print that development has moved to Github? I had no idea this repo existed without doing a lot of digging- CodePlex is still one of the top hits on Google.

**Comment by hugener** [comment]
I posted a comment on what appears to be the last remaining bug:
https://github.com/gsscoder/commandline/issues/361
I seems to be clear what's wrong, but I'm not sure if there is an easy fix.

**Comment by tesharp** [comment]
Any chance to create a release package soon? Its been a year and looking at NuGet it seems a lot of people have just resolved to using the unofficial package by now.. Minor bugs can be fixed later as well. If the goal is to have a bug free release then I guess we have to wait another year.
